form.js

```diff
@@ -1,26 +1,32 @@
-(function () {
-  "use strict";
-  const root = document.getElementById("root");
-  const form = document.createElement("form");
+const root = document.getElementById("root");
+const form = createNewEl('form');
 
+function createNewEl(el){
+  return document.createElement(el);
+}
 
-  const inputEmail = document.createElement("input");
-  inputEmail.type = "text";
-  inputEmail.placeholder = "Enter your email";
+function createNewAttr(el, atr, val){
```

Comment:
It doesn't create something new. It adds an attribute. Get the object as a second parameter.
atr -> attr